lib/vsc/utils/mail.py

```diff
@@ -115,8 +115,6 @@ def _connect(self):
             context = ssl.create_default_context()
             s.starttls(context=context)
             logging.debug("Started TLS connection")
-        else:
-            s.connect()
```

Comment:
I distinctly recall this was needed at some point.

Reply:
@itkovian it's indeed still needed if the mail host is not specified.
I was wrong thinking that it must always be specified.
fixed in https://github.com/hpcugent/vsc-base/pull/331/commits/1b9ff71f551fd877f7673fb4f99f5228e315ecf0